src/accelerate/test_utils/scripts/test_script.py

```diff
@@ -541,6 +541,20 @@ def test_split_between_processes_tensor():
     state.wait_for_everyone()
 
 
+def test_breakpoint():
```

Comment:
```suggestion
def test_trigger():
```

Or possible: `test_flag_tensor`. If renamed, also rename it further below.